server/src/controllers/Events.js

```diff
@@ -40,13 +40,13 @@ module.exports = {
           public_id: image.public_id
         };
       }
-      res.json({
+      return res.status(200).json({
         id: event._id
       });
     } catch (err) {
-      res.status(203).send({
-        err: err
-      });
+        return res.status(500).json({
+          	message: err.message
```

Comment:
Since 500 is a server side error, it'd be great if you could replace the key with "err" or "error" to signify that this is, in fact, an error. Also, could you explain why 500 is the appropriate code here ?

Thanks.

Reply:
Alright